Scripts/AzureAppConfig/compare_aac_exports.py

```
@@ -237,6 +270,18 @@ def print_results(results: Dict[str, List]):
 
+    # Tag-only changes
+    if results['tags_changed']:
+        print(f"\nTAG CHANGES ONLY: {len(results['tags_changed'])}")
+        print("-" * 80)
+        for item in results['tags_changed']:
+            label_str = f" [Label: {item['label']}]" if item['label'] else " [No Label]"
+            print(f"\n  Key: {item['key']}{label_str}")
+            for change in item['tag_changes']:
+                print(f"    {change}")
+    else:
+        print(f"\nTAG CHANGES ONLY: 0")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)